src/modules/70-pipeline/pages/execution/ExecutionTestView/__test__/BuildTestsApp.test.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React from 'react'
+import { render, waitFor } from '@testing-library/react'
+import { defaultAppStoreValues } from '@common/utils/DefaultAppStoreData'
+import * as ExecutionContext from '@pipeline/context/ExecutionContext'
+import { TestWrapper } from '@common/utils/testUtils'
+import * as tiService from 'services/ti-service'
+import TestSuiteMock from './mock/reports-test-suites.json'
+import TestCaseMock from './mock/reports-test-cases.json'
+import TotalTestsZeroMock from './mock/total-tests-zero.json'
+import InfoMock from './mock/info.json'
+import CallGraphMock from './mock/callgraph.json'
+import BuildsMock from './mock/builds.json'
+import mockContext from './mock/context.json'
+import mockParams from './mock/params.json'
+import BuildTestsApp from '../BuildTestsApp'
+
+describe('BuildTestsApp', () => {
+  test('Initial render with TI_MFE_ENABLED enabled', async () => {
+    jest.mock('@pipeline/context/ExecutionContext', () => ({
+      useExecutionContext: jest.fn().mockReturnValue(() => mockContext)
```

Comment:
I don't think you need the `() =>` here. `mockReturnValue` creates a function that will return whatever you give it. In this case it will return a function that, when called, will return a function that can be called to get `mockContext`.
```suggestion
      useExecutionContext: jest.fn().mockReturnValue(mockContext)
```